### PR TITLE
docs(readme): explain where to get the token from

### PR DESCRIPTION
**Pull request by lirantal**
I received a ping on where to find the Snyk token in order to use the actions so I'm updating the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,3 +61,17 @@ jobs:
 See the individual Actions linked above for per-language instructions.
 
 Note: GitHub Actions will not pass on secrets set in the repository to forks being used in pull requests, and so the Snyk actions that require the token will fail to run.
+
+## Getting your Snyk token
+
+The Actions example above refer to a Snyk API token:
+
+```
+      env:
```
Comment (garethr) [inline README.md:70]: If you could remove the extra whitespace here.

File: README.md [hunk 2]
```diff
@@ -61,3 +61,17 @@ jobs:
 See the individual Actions linked above for per-language instructions.
 
 Note: GitHub Actions will not pass on secrets set in the repository to forks being used in pull requests, and so the Snyk actions that require the token will fail to run.
+
+## Getting your Snyk token
+
+The Actions example above refer to a Snyk API token:
+
+```
```
Comment (garethr) [inline README.md:69]: Add the yaml suffix for highlighting, so it matches the other examples. eg:

```yaml

File: README.md [hunk 3]
```diff
@@ -61,3 +61,17 @@ jobs:
...
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+```
+
+Every Snyk account has this token, and you can find it in one of two ways:
+1. If you're using the [Snyk CLI](https://support.snyk.io/hc/en-us/articles/360003812458-Getting-started-with-the-CLI) you can retrieve it by running `snyk config get api`
```
Comment (garethr) [inline README.md:75]: Missing a full spot. The second list item has one. Best to be consistent. Either add a full stop on this line or remove from the one below

**Comment by lirantal** [comment]
thanks @garethr, good catches. I updated all of them.
Let me know if something else is missing.

**Comment by garethr** [comment]
Thanks @lirantal.

**Pull request merged**